libraries/GCS_MAVLink/GCS_Common.cpp

```diff
@@ -1132,7 +1142,9 @@ void GCS_MAVLINK::update_send()
         AP::logger().handle_log_send();
     }
 #endif
-    send_ftp_replies();
+    if (!is_high_latency_link) {
+        send_ftp_replies();
+    }
```

Comment:
We shouldn't ever have FTP replies to send!

Implications here (and for the parameter stuff above) is that we may allocate resources that are never freed (or are slow to free or something).

Reply:
Removed